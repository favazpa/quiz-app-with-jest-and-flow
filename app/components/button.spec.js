// @flow
import React from 'react';
import {Component} from 'react';
import Button from './button';
import ShallowRenderer from 'react-test-renderer/shallow';

let renderer = new ShallowRenderer();

describe('src/component/button', () => {
  const renderButton = () => {
    renderer.render(<Button />);
    return renderer.getRenderOutput();
  };

  test('should render correctly', () => {
    const result = renderButton();
    expect(result).toMatchSnapshot();
  });
});