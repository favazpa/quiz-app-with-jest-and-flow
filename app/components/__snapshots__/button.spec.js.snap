// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/component/button should render correctly 1`] = `
<TouchableOpacity
  style={
    [
      {
        "backgroundColor": "green",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "white",
        },
        undefined,
      ]
    }
  />
</TouchableOpacity>
`;
